src/components/ReportActionItem/ReportActionItemImage.js

```diff
@@ -79,4 +107,8 @@ ReportActionItemImage.propTypes = propTypes;
 ReportActionItemImage.defaultProps = defaultProps;
 ReportActionItemImage.displayName = 'ReportActionItemImage';
 
-export default ReportActionItemImage;
+export default withOnyx({
```

Comment:
What's the benefit of that versus connecting to ONYX here?
I feel like passing this image data
```js
{image: 'eReceipt/FAKE_2', thumbnail: '', transactionID: 'FAKE_2'},
```
 makes more sense than passing it all like:
```js
{image: 'eReceipt/FAKE_2', thumbnail: '', transaction: {
        transactionID: 'FAKE_3',
        amount: 1000,
        currency: 'USD',
        cardID: 5,
        merchant: 'United Airlines',
        mccGroup: 'Commuter',
        created: '2023-07-24 13:46:20',
        hasEReceipt: true,
    },},
```

but also I'm totally fine to do it that way if theres a reason for it!

Reply:
Only reason I can think is that `withOnyx` would delay rendering the component until the data is fetched. Not a big deal in this case since the data set is small and most likely in memory, but since we already have the data in the parent component we could just pass it as a prop to avoid reading it again and also clean up the HOC logic

Reply:
Updated!